### ISSUE TITLE
Unknown config attribute `params.genomes.GRCh37.salmon_index`

### ISSUE THREAD

**Issue by winni2k** [nf-core/scrnaseq#29]
When I run the following command:
```bash
export NXF_OPTS='-Xms1g -Xmx4g'
nextflow run -r 1.0.0 nf-core/scrnaseq\
          --reads 'data/H204SC19123229/raw_data/*/*_R{{1,2}}_001.fastq.gz'\
          --aligner kallisto\
          --type 10x\
          --chemistry V3\
          --genome GRCh37\
          --outdir 'results/expression_counts'\
          -profile singularity
```
I get the following error:
```
N E X T F L O W  ~  version 19.10.0
Launching `nf-core/scrnaseq` [sharp_shaw] - revision: 884e541285 [1.0.0]
Unknown config attribute `params.genomes.GRCh37.salmon_index` -- check config file: ...
```

I have tried changing the `-r` argument to `dev`, using `--genome GRCh38`, and supplying my own `--gtf` file, but I always get this error.

I'm not trying to use salmon?

**Comment by apeltzer** [comment]
Hi @winni2k!

We're currently not shipping the Salmon index with iGenomes (something we'd love to do but do not do at the moment). In the Salmon case, the lookup of the required genome files in iGenomes therefore fails. You should be able to get around this by supplying both the `--fasta` and `--gtf` parameters and _not_ the `--genome` one. Then the pipeline builds the indices on demand for you. You may later then reuse the salmon index built supplying it with `--salmon_index ...`

Let me know if that helps :+1:

**Comment by grst** [comment]
This issue is likely fixed in the 2.1.0 release of the pipeline. If not, please open a new issue.
